flipper/contrib/postgresql.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2018 eShares, Inc. dba Carta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from contextlib import contextmanager
+from typing import Iterator, Optional
+
+from psycopg import Connection, connect, sql
+
+from .interface import AbstractFeatureFlagStore, FlagDoesNotExistError
+from .storage import FeatureFlagStoreItem, FeatureFlagStoreMeta
+from .util.date import now
+
+
+class PostgreSQLFeatureFlagStore(AbstractFeatureFlagStore):
+    def __init__(
+        self,
+        conninfo: str,
+        table_name: str = "feature_flags",
+    ) -> None:
+        self._conninfo = conninfo
+        self._table_name = sql.Identifier(table_name)
+        self._name_column = sql.Identifier("name")
+        self._item_column = sql.Identifier("item")
+        self._run_migrations()
```

Comment:
I would love for this to be optional, in case the user wants to manage the schema manually. I could also see edge cases happening where, for some reason, we try to initialize this before postgres is available.